dts/bindings/sent/sent-controller.yaml

```diff
@@ -0,0 +1,92 @@
+# Copyright 2025 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Single Edge Nibble Transmission (SENT) Controller
+
+include: base.yaml
+
+properties:
+  reg:
+    required: true
+
+child-binding:
+  description: |
+    Each child node defines the configuration of a Single Edge Nibble Transmission (SENT) channel
+
+  properties:
+    reg:
+      type: int
+      required: true
+      description: Channel identifier.
+
+    num-data-nibbles:
```

Comment:
Is this something we would want to configure at runtime, or will it only be configured at initialization?

Reply:
As the specification recommends that SENT receiver allows for up to at least 8 data nibbles to ensure forward compatibility with future protocol revisions.